src/modules/base/Module.sol

```diff
@@ -87,6 +92,22 @@ abstract contract Module is IModule, Initializable, ContextUpgradeable {
         _;
     }
 
+    /// @dev same function as OZ initializer, but for the init2 function
+    modifier initializer2() {
+        if (__Module_initialization == true) {
```

Comment:
This looks like it could be simplified slightly - or is this on purpose? Haven't found any other instance of this in our codebase though :D